test/zdtm/static/cgroup_threads.hook

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+[ "$1" == "--clean" -o "$1" == "--pre-restore" ] || exit 0
+
+set -e
+
+tname=$(mktemp -d cgclean.XXXXXX)
+mount -t cgroup none $tname -o "none,name=zdtmtst"
+
+echo "Cleaning $tname"
+
+set +e
+rmdir "$tname/subcg_threads/subsubcg/"
+rmdir "$tname/subcg_threads/"
```

Comment:
You can do `rmdir ... || true` instead of temporarily disabling `set -e`.

Reply:
Also, all the directories can be removed by a single `rmdir`.

Reply:
Done. But leaved two instruction to make intentions clear